src/data/constants/learnerCredit.js

```diff
@@ -0,0 +1,28 @@
+export const PAGE_TITLE = 'edX Enterprise Catalogs';
+
+/**
+ * Event names to use for tracking
+ * Spec: https://openedx.atlassian.net/wiki/spaces/AN/pages/2624455676/Enterprise+public+catalog+Event+Design
```

Comment:
This linked doc is specific to public catalog; doesn't really make sense to link to it from admin portal?